pkg/apis/pipeline/v1beta1/task_conversion_test.go

```diff
@@ -326,3 +327,81 @@ spec:
 		})
 	}
 }
+
+func TestTaskConversionFromDeprecated(t *testing.T) {
+	tests := []struct {
+		name string
+		in   *v1beta1.Task
+		want *v1beta1.Task
+	}{{
+		name: "input resources",
```

Comment:
do we need 2 cases here or can we merge them into 1?

Reply:
We can definitely merge them into one. I think keeping them separate is cleaner for readability. One handling inputs and the other, outputs.

Reply:
That's ok! I don't have strong opinions here. 😄